<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Crop_dialog</class>
 <widget class="QDialog" name="Crop_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>211</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>403</width>
    <height>211</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>403</width>
    <height>211</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size: 23px;</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>119</y>
     <width>162</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Horizontal crop</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>54</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical crop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>166</y>
     <width>151</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(75, 75,75);
color: white;</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>166</y>
     <width>131</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(75, 75,75);
color: white;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="vertical_crop">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>80</y>
     <width>149</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="horizontal_crop">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>120</y>
     <width>149</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>363</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>8</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font-size: 23px;</string>
      </property>
      <property name="text">
       <string>Crop direction</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="crop_direction">
      <property name="styleSheet">
       <string notr="true">font-size: 23px;</string>
      </property>
      <item>
       <property name="text">
        <string>Center</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Upper left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Upper right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lower right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lower left</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>crop_direction</tabstop>
  <tabstop>vertical_crop</tabstop>
  <tabstop>horizontal_crop</tabstop>
  <tabstop>ok_button</tabstop>
  <tabstop>cancel_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
